VRDR/DeathRecord.cs

```diff
@@ -4034,13 +4120,17 @@ public string BirthRecordId
                     BirthRecordIdentifier.Status = ObservationStatus.Final;
                     BirthRecordIdentifier.Code = new CodeableConcept("http://terminology.hl7.org/CodeSystem/v2-0203", "BR", "Birth registry number", null);
                     BirthRecordIdentifier.Subject = new ResourceReference("urn:uuid:" + Decedent.Id);
-                    BirthRecordIdentifier.Value = new FhirString(value);
                     AddReferenceToComposition(BirthRecordIdentifier.Id);
                     Bundle.AddResourceEntry(BirthRecordIdentifier, "urn:uuid:" + BirthRecordIdentifier.Id);
                 }
+                if (!String.IsNullOrWhiteSpace(value))
+                {
+                    BirthRecordIdentifier.Value = new FhirString(value);    
+                }
                 else
                 {
-                    BirthRecordIdentifier.Value = new FhirString(value);
+                    // TODO need to check for an existing data absent reason in the fhir
```

Comment:
As per the comment, I think you need to check for the existence (or simply delete all existing ones) before creating a new one.

Reply:
I looked into this more and it will be fine as is. The only time we need to use that is when we are converting between IJE and FHIR and need to create the data absent field. If we're just reading in fhir and using fhir then the data will be there - I ran a couple CLI json2json and checkJson tests to verify. I'll remove the todo comment